.remarkignore

```diff
@@ -1 +1 @@
-CHANGELOG.md
+
```

Comment:
`CHANGELOG.md` is generate by using `conventional-changelog` based on the angular configuration.

We noticed that `remark` isn't quite happy with theses generated files so that's the reason of why we have added them to the `.remarkignore` file.

IMO we should stay with the actual configuration.

Reply:
build fails with lot of errors like "CHANGELOG.md file is ignored", its not possible to commit our changes with this. Is there any other way?

Reply:
i undo my changes